src/containers/BooksList.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+const BooksList = ({ books }) => (
+  <div>
+    <table>
+      <thead>
+        <tr>
+          <th>ID</th>
+          <th>Title</th>
+          <th>Category</th>
+        </tr>
+      </thead>
+      {books.map(book => (
```

Comment:
We need to import `Book Component`.
`BookList` should displays books list in html table using `Book component`

Example:
`{books.map(book => (<Book book={book} key={book.id} />))}`

Reply:
Changes applied,
Book component is now imported into BooksList component and rendering properly.